src/app/game/core/actions/harvestAction.ts

```
@@ -0,0 +1,142 @@
+export const harvest = (
+  field: FieldType,
+  discardPile: CardsType[],
+): { field: FieldType; harvestMoney: number; discardPile: CardsType[] } => {
+  const newField = { ...field };
+
+  if (newField.crops === null) {
+    console.log("no crops to harvest");
+    return { field, harvestMoney: 0, discardPile };
+  } else {
+    const harvestMoney = findThePrice(newField.crops, newField.manure) ?? 0;
+    const harvestMoney = findThePrice(newField.crops, newField.manure) ?? 0;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'harvestMoney'. Consider to delete it or rename it.

'harvestMoney' is defined here:


(lint/suspicious/noRedeclare)